src/promptflow/promptflow/storage/_run_storage.py

```diff
@@ -30,6 +30,14 @@ def persist_flow_run(self, run_info: FlowRunInfo):
         raise NotImplementedError("AbstractRunStorage is an abstract class, no implementation for persist_flow_run.")
 
 
+class BatchRunStorage(AbstractRunStorage):
+    def load_node_run_info(self, line_number) -> List[NodeRunInfo]:
```

Comment:
Please rename to `load_node_run_infos_of_line` for better clarity.

Reply:
To reuse the code in load_detail, I divided the logic into two parts, "load_node_run_info" and "get_node_run_info_by_line_number". Is this solution feasible?
![1707104363704](https://github.com/microsoft/promptflow/assets/39176492/02570834-de9f-484b-a70d-f8d7e2678d18)

Reply:
Changed to AbstractBatchRunStorage.